vars/govuk.groovy

```diff
@@ -682,24 +732,23 @@ def pushTag(String repository, String branch, String tag) {
  * @param deployTask Deploy task (deploy, deploy:migrations or deploy:setup)
  */
 def deployIntegration(String application, String branch, String tag, String deployTask) {
```

Comment:
I guess we left the `branch` argument for backwards compatibility, but I wonder if it's worth adding a message or comment about how it's deprecated?

I can only find 4 non-archived repos which call it directly, maybe it's not worth keeping it around? https://github.com/search?q=org%3Aalphagov+deployIntegration&type=code

Reply:
Interesting point. I'd not appreciated the smell that redundant argument leaves. 

How about I rename it to deployToIntegration without that param and keep deployIntegration as a deprecated function that calls deployToIntegration?

Reply:
Yeah, that sounds like a reasonable way forward!